swarm/storage/hasherstore.go

```diff
@@ -243,7 +269,11 @@ func (h *hasherStore) newDataEncryption(key encryption.Key) encryption.Encryptio
 
 func (h *hasherStore) storeChunk(ctx context.Context, ch Chunk) {
 	atomic.AddUint64(&h.nrChunks, 1)
```

Comment:
no longer needed, also the field on the struct is no longer used as of such

Reply:
ok. done